missing-kit-form/form.tex

```diff
@@ -0,0 +1,41 @@
+\documentclass[a4paper, 12pt]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage[default,osfigures]{opensans}
+
+\usepackage[margin=0.5in]{geometry}
+\setlength{\parindent}{0cm}
+
+\pagenumbering{gobble}
+
+\begin{document}
+
+\begin{center}
+
+    {\huge{Student Robotics Missing Kit Form}}
+
+    \medskip
+
+\end{center}
+
+Please fill out this form if some of the team's kit is missing when they return it at the end of the event. The required parts can be found on the "Kit Contents List". \\
+
+\bigskip
+
+{\large{Team TLA}: \hrulefill} \\
+
+{\large{Volunteer Name}: \hrulefill} \\
+
+{\large{Missing Parts}:}
+
+\vspace*{\fill}
+
+I, the team leader for the above team, agree that the above listed parts are missing. \\
```

Comment:
What do we want the team-leader to do after this? Are they going to undertake to find and return the item? (I realise that only applies to some reasons for the kit being missing) Is there something we can put here to log their next action?

Reply:
If they're going to look for and return the missing items, are we going to give the team leaders a copy of the list of missing items? Or perhaps email them afterwards with a list? Maybe get them to take a photo of this form for their own reference?

Reply:
I suspect sending an email with the information in would be ideal. I'll add that to the runbook.